pkg/controller/infrastructure/templates/main.tpl.tf

```diff
@@ -4,12 +4,6 @@ provider "alicloud" {
   region = "{{ .alicloud.region }}"
 }
 
-// Import an existing public key to build a alicloud key pair
-resource "alicloud_key_pair" "publickey" {
```

Comment:
I also believe that the infrastructure integration test has to be adapted. Currently the test should have expectations that the ECS key pair exists. I don't see any changes in the integration test as part of this PR. Can you please check?

Reply:
Yes. will check it.